### DISCUSSION TITLE
Leaderboards with bad score ordering

### DISCUSSION THREAD

**Discussion by Walavouchey** [ppy/osu#24227] [Q&A]
was told this was [mentioned](https://www.youtube.com/watch?v=bocgaVISSfg&t=1744s) in the meeting chat right when the issue was brought up

[Kurokotei - Galaxy Collapse [Galactic]](https://osu.ppy.sh/beatmapsets/396221#osu/862088)

**Comment by WitherFlower** [comment]
Here are the main issues of scoring in its current state. Hope this helps giving somewhat of a path forward.

Issues with scorev1 :
- FCs aren't ordered by acc because acc depends on where you dropped 100s (aka "getting scorev1'd")
    - Cases where it's noticeable : *literally* every leaderboard in the game. Just press F2 and you'll find a case of this
    - This is solved by SV2

- Combo has too much importance compared to acc
    - Cases where it's noticeable : leaderboards with few or no FCs, like <https://osu.ppy.sh/beatmapsets/396221#osu/862088> (see n°1 vs n°2 on the map), and tournaments 
    - This is solved by SV2

- Missing in the middle of the map is too punishing
    - Cases where it's noticeable : this one is mostly felt by experienced tournament players / players that use SV2 a lot. I suggested in the meeting's chat that we should ask tournament players how they feel about this to make sure it's a good direction.
    - This is solved by #24166

Issues with standardised scoring :
- Bonus portion depends on the number of bonus objects in the map
    - Cases where it's noticeable : marathon maps with a lot of spinners, like this one <https://osu.ppy.sh/beatmapsets/121591#osu/311269>, where the bonus score can reach above 100k points in current lazer scoring
    - This is currently not addressed but a solution was debated earlier in this channel

**Reply by Zyfarok** [reply]
to "Issues with standardised scoring" I would also add:
- slider-ends have way too little value. They had much more impact in scorev1 and v2 since they gave 1 combo and not aiming them properly would turn the slider into a 100. There value is currently 10 in lazer but should probably be at least 100, if not 200. (By comparison, slider-ticks only give 30 points, but they give 1 combo which makes them much more important)
This issue is being tracket here: https://github.com/ppy/osu/issues/23403

It's not directly about the scoring formula, but it's highly related.
